src/main/app/claims/models/claimStatus.ts

```diff
@@ -20,6 +20,8 @@ export enum ClaimStatus {
   CLAIMANT_ACCEPTED_ADMISSION_AND_REQUESTED_CCJ,
   ADMISSION_SETTLEMENT_AGREEMENT_REACHED,
   CLAIMANT_REJECTED_DEFENDANT_AS_BUSINESS_RESPONSE,
+  CLAIMANT_REJECTED_DEFENDANT_AS_BUSINESS_RESPONSE_WITH_MEDIATION,
```

Comment:
I question we require new statuses. Please look at comments.

Reply:
Tested with FreeMediationOption.YES and everything works fine, however, if I remove the additional statuses the correct content doesn't show, therefore I think we need the extra statuses

Reply:
I agree with surjeet. These statuses are using the same content as other statuses and shouldn't be needed. You should just need to update hasClaimantRejectedPartAdmission if the status is not showing up

Reply:
Ok let me have a look